.github/workflows/ci.yml

```diff
@@ -256,6 +256,7 @@ jobs:
       - run: python -m pytest -n 2 modin/test/test_partition_api.py
       - run: python -m pytest modin/test/test_utils.py
       - run: python -m pytest asv_bench/test/test_utils.py
+      - run: python -m pytest modin/test/exchange/dataframe_protocol/base
```

Comment:
What's the point of `exchange/dataframe_protocol`? What other directories could possible be in `exchange/...`?